employee-rostering-frontend/src/store/employee/employee.test.ts

```diff
@@ -0,0 +1,219 @@
+/*
+ * Copyright 2019 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { mockStore } from '../mockStore';
+import { AppState } from '../types';
+import * as actions from './actions';
+import reducer, { employeeOperations } from './index';
+import {withElement, withoutElement, withUpdatedElement} from 'util/ImmutableCollectionOperations';
+import {onGet, onPost, onDelete, resetRestClientMock} from 'store/rest/RestTestUtils';
+import Employee from 'domain/Employee';
+
+describe('Employee operations', () => {
+  it('should dispatch actions and call client', async () => {
+    const { store, client } = mockStore(state);
+    const tenantId = store.getState().tenantData.currentTenantId;
+    
+    const mockEmployee: Employee = {
+      tenantId: tenantId,
```

Comment:
What is the actual tenant ID? It would be better if it was a constant *defined* by the test, independent on some default values. It's probably zero but would the test work if we used a non-default value?

Reply:
I defined it in the test; the initial state is defined at the `const state = {...}` at the bottom of the test.

Reply:
Thanks. I missed that `mockStore(state)` accepts the initial state.